src/Shared/ReuseableStringBuilder.cs

```diff
@@ -17,19 +17,13 @@ namespace Microsoft.Build.Shared
     /// <remarks>
     /// You can add any properties or methods on the real StringBuilder that are needed.
     /// </remarks>
-    internal sealed class ReuseableStringBuilder : IDisposable, IInternable
```

Comment:
Ideally this class should be subsumed by the RopeBuilder right? Worth adding a class comment saying that?

Reply:
Done, thanks!